### PR TITLE
fix: use last CLI api

### PR DESCRIPTION
**Pull request by moul**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ inputs:
     description: 'Relative path to the documentation file'
     required: true
     default: api-contract.yml
-  id:
```
Comment (scharrier) [inline action.yml:8]: Removing the id will generate a GitHub action error here, for those who currently use the action, until they update the call by replacing `id` per `doc`.

I think that it's acceptable as for now we do not version this tool (and we do not have a lot of customers using this action at the moment), but once this is done we should clearly start tagging versions and point the README example and the action example to the latest tag.

WDYT @antlio?

**Comment by scharrier** [comment]
Thanks a lot for this PR @moul. As there's a [breaking change in it](https://github.com/bump-sh/github-action/pull/2#pullrequestreview-439955839) (and we did not versioned the action) we just need to double check that we won't break any of our customers CI before merging it.

File: action.yml (hunk 1 above)
Comment (moul) [reply action.yml:8]: another option is to support both `id` and `doc` as optional parameters and only the good one dynamically

Comment (moul) [reply action.yml:8]: Also I think that we can also just break the compatibility since the current version (0.1) is broken and make the CI fail anyway :)

Comment (antlio) [reply action.yml:8]: I agree with the last @moul comment, which join your first assumption @scharrier 😋

File: Dockerfile [hunk 2]
```diff
@@ -1,3 +1,4 @@
 FROM ruby:2.6
+RUN gem install bump-cli
```
Comment (scharrier) [inline Dockerfile:2]: Gem installation is done directly in the `entrypoint.sh` file ([see here](https://github.com/bump-sh/github-action/blob/master/entrypoint.sh#L4)), we shouldn't need to add it here.

**Comment by scharrier** [comment]
Sorry for the delay @moul, this week has been crazy for us. We can merge this PR as soon as https://github.com/bump-sh/github-action/pull/2#pullrequestreview-441543031 is fixed: we are OK to break compatibility on our master branch, we will then start releasing versions.

Thanks a lot for your help here 🙏

File: Dockerfile (hunk 2 above)
Comment (moul) [reply Dockerfile:2]: It's not a duplicate, I removed it from the entrypoint.sh, it's just a move to the expected location

If you really prefer not placing deps install in the Dockerfile, I can move it back to the entrypoint.sh and add a comment pointing to this issue for other people that don't understand why it was done in that way :) up to you

Comment (scharrier) [reply Dockerfile:2]: Well, moving the `gem install` at the `Dockerfile` level seems to change the behavior: we now get Ruby version issues, like you can see here: https://github.com/bump-sh/bump-ci-example/runs/840628727?check_suite_focus=true

I did the same test but keeping the `gem install` inside the `entrypoint.sh` script, and this time it worked: https://github.com/bump-sh/bump-ci-example/runs/840555890?check_suite_focus=true

I don't know what's happening, but I propose to keep it the way it works for now, we'll iterate on this later. Works for you?

**Comment by moul** [comment]
I reverted the "move install to Dockerfile" and added a comment that links to this PR thread, can you recheck a last time?

**Comment by scharrier** [comment]
I just tested it again, it works. https://github.com/bump-sh/bump-ci-example/runs/844627777

Can you just edit the commit comment to have something accurate with its content? I'll then merge it :)

**Comment by moul** [comment]
done

**Comment by scharrier** [comment]
Thans for your time @moul 🙏 We will release a working `0.2` version soon.

**Pull request merged**